test/backlog/load_error_test.rb

```diff
@@ -1,5 +1,6 @@
 require 'test_helper'
 
 class LoadErrorTest < Minitest::Test
+	include HumanistErrorsSupport
```

Comment:
I try to stick to the ruby standard 2 space for indentation, could you update the white space here?